src/scss/components/contact.scss

```diff
@@ -0,0 +1,43 @@
+.contactUs {
+  @extend .container;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-left: 19px;
+  padding-right: 19px;
+
+  &__title {
+    display: flex;
```

Comment:
Remove the flex, so that the `text-align: center;` property can be applied correctly